provision/kubernetes/deploy.go

```diff
@@ -928,6 +928,43 @@ func monitorDeployment(ctx context.Context, client *ClusterClient, dep *appsv1.D
 	return revision, nil
 }
 
+type KubernetesInternalAddress struct {
+	Domain   string
+	Protocol apiv1.Protocol
+	Port     int32
+}
+
+func (m *serviceManager) KubernetesInternalAddresses(ctx context.Context, a provision.App) ([]*KubernetesInternalAddress, error) {
+	ns, err := m.client.AppNamespace(a)
+	if err != nil {
+		return nil, err
+	}
+	addresses := []*KubernetesInternalAddress{}
+
+	// TODO: discover app processes
```

Comment:
You can use the App custom resource definition to get this

Reply:
Thank you so very much! @andrestc